core/templates/css/oppia.css

```diff
@@ -2527,8 +2527,570 @@ oppia-parameter {
   pointer-events: none;
 }
 
-.oppia-disabled-image-link {
-  pointer-events: none;
+/* Improvements tab */
```

Comment:
Please let's not try to add more stuff to the (already overloaded) oppia.css. Can we have these styles within the directives that use them?